app-backend/database/src/main/scala/com/azavea/rf/database/Database.scala

```diff
@@ -20,6 +20,7 @@ class Database(jdbcUrl: String, dbUser: String, dbPassword: String) {
   hikariConfig.setJdbcUrl(jdbcUrl)
   hikariConfig.setUsername(dbUser)
   hikariConfig.setPassword(dbPassword)
+  hikariConfig.setLeakDetectionThreshold(3000L)
```

Comment:
This is fine then. Just seemed weird since the docs reference numbers that look like integers without specific types.